### ISSUE TITLE
what's meaning of margin value instead of transformed prediction

### ISSUE THREAD

**Issue by fword** [Microsoft/LightGBM#1808]
what's meaning of margin value instead of transformed prediction

**Comment by guolinke** [comment]
for binary classification, it is the logits before sigmoid.
For multi-class, it is the logits berfore softmax.
